app/templates/logos.hbs

```diff
@@ -1,175 +1,192 @@
-<h1>Branding</h1>
+<div class="container">
+  <h1>Branding</h1>
 
-<p>
-  The Ember brand and associated assets are part of a meticulously curated experience. We take great pride in what our team and community have helped bring to life, and ask that you follow our guidelines to ensure consistency across all Ember properties.
-</p>
+  <p>
+    The Ember brand and associated assets are part of a meticulously curated experience. We take great pride in what our team and community have helped bring to life, and ask that you follow our guidelines to ensure consistency across all Ember properties.
+  </p>
 
-<p><a href="/images/brand/Ember-Brand-Guidelines.pdf">Download brand guidelines (PDF)</a></p>
+  <p><a href="/images/brand/Ember-Brand-Guidelines.pdf">Download brand guidelines (PDF)</a></p>
 
```

Comment:
Is there a `<section>` that should wrap this h2+content? There is a `<section>` element that wraps the next h2.

Reply:
I have wrapped it in a section and added the aria-labelledby 👍